tests/exporters/test_export.py

```diff
@@ -80,8 +81,9 @@ def _get_models_to_test(export_models_dict: Dict, random_pick: Optional[int] = N
                     (f"{model_type}_{task}", model_type, model_name, task, neuron_config_constructor)
                 )
 
+    random_pick = os.environ.get("MAX_EXPORT_TEST_COMBINATIONS", None)
     if random_pick is not None:
-        return sorted(random.choices(models_to_test, k=random_pick))
+        return sorted(random.choices(models_to_test, k=int(random_pick)))
```

Comment:
Are the CIs running all tests now? Did you set `MAX_EXPORT_TEST_COMBINATIONS`? I wonder if it would take too long.

Reply:
I used it while debugging the tests, but eventually I ran all of them because if I set it to just 1 it runs just ... one test.
We would need to figure out a compromise for these tests. Maybe have a few quick steps with just one test to catch early failures, then the whole list that takes tens of minutes.